enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -120,27 +120,40 @@ func getOrCreateContainerImage(ctx context.Context, containerImage string) (stri
 // NB: We use docker because it's already installed - but podman can also be
 // used interchangeably and does not require root access.
 func convertContainerToExt4FS(ctx context.Context, containerImage string) (string, error) {
-	rootFSDir, err := os.MkdirTemp("", "containerfs-*")
+	// Make a temp directory to work in. Delete it when this fuction returns.
+	rootUnpackDir, err := os.MkdirTemp("", "container-unpack-*")
 	if err != nil {
 		return "", err
 	}
-	defer os.RemoveAll(rootFSDir)
+	defer os.RemoveAll(rootUnpackDir)
```

Comment:
nit: would it make sense to log anything if this fails?

Reply:
I think it's overkill tbh -- given that it's a tmpdir and if removing a directory fails, we've got other problems